fastlane/lanes/release.rb

```diff
@@ -0,0 +1,236 @@
+    setfrozentag(repository: GHHELPER_REPO, milestone: version, freeze: false)
+    create_new_milestone(repository: GHHELPER_REPO)
+    close_milestone(repository: GHHELPER_REPO, milestone: version)
+
+    # Start the build
+    trigger_release_build(branch_to_build: "release/#{version}")
+  end
+
+  # Triggers a beta build on CI
+  #
+  # @option [String] branch_to_build The name of the branch we want the CI to build, e.g. `release/19.3`
+  #
+  lane :trigger_beta_build do |options|
+    trigger_buildkite_release_build(branch: options[:branch_to_build], beta: true)
+  end
+
+  # Triggers a stable release build on CI
+  #
+  # @option [String] branch_to_build The name of the branch we want the CI to build, e.g. `release/19.3`
+  #
+  lane :trigger_release_build do |options|
+    trigger_buildkite_release_build(branch: options[:branch_to_build], beta: false)
+  end
+end
+
+#################################################
+# Helper Functions
+#################################################
+
+# Triggers a Release Build on Buildkite
+#
+# @param [String] branch The branch to build
+# @param [Boolean] beta Indicate if we should build a beta or regular release
+#
+def trigger_buildkite_release_build(branch:, beta:)
+  buildkite_trigger_build(
+    buildkite_organization: 'automattic',
+    buildkite_pipeline: 'wordpress-ios',
+    branch: branch,
+    environment: { BETA_RELEASE: beta },
+    pipeline_file: 'release-builds.yml'
+  )
+end
+
+# Checks that the Gutenberg pod is reference by a tag and not a commit
+#
+desc 'Verifies that Gutenberg is referenced by release version and not by commit'
+lane :gutenberg_dep_check do
+  res = ''
+
+  File.open File.join(PROJECT_ROOT_FOLDER, 'Podfile') do |file|
+    res = file.find { |line| line =~ /^(?!\s*#)(?=.*\bgutenberg\b).*(\bcommit|tag\b){1}.+/ }
+  end
+
+  UI.user_error!("Can't find any reference to Gutenberg!") if res.empty?
+  if res.include?('commit')
+    UI.user_error!("Gutenberg referenced by commit!\n#{res}") unless UI.interactive?
+
+    unless UI.confirm("Gutenberg referenced by commit!\n#{res}\nDo you want to continue anyway?")
+      UI.user_error!('Aborted by user request. Please fix Gutenberg reference and try again.')
+    end
+  end
+
+  UI.message("Gutenberg version: #{(res.scan(/'([^']*)'/))[0][0]}")
+end
+
+# Returns the path to the extracted Release Notes file for the given `app`.
+#
+# @param [String|Symbol] app The app to get the path for, must be one of `wordpress` or `jetpack`
+#
+def extracted_release_notes_file_path(app:)
+  paths = {
+    wordpress: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt'),
+    jetpack: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Jetpack', 'Resources', 'release_notes.txt')
+  }
+  paths[app.to_sym] || UI.user_error!("Invalid app name passed to lane: #{app}")
+end
+
+# Prints the message to remind the Release Manager to audit the Release Notes after code freeze.
```

Comment:
I'm probably the one who wrote this in the first place, but here's a suggestion to make it more succinct:

```suggestion
# Prints a reminder to audit the Release Notes after code freeze.
```